### PR TITLE
Feature/UI design

### PR DESCRIPTION
**Pull request by sathish199308**

### CODE REVIEW

File: BookingSystem/Views/Dashboard/DashboardView.cshtml [hunk 1]
```diff
@@ -118,6 +118,18 @@
                             <i class="fas fa-mobile"></i>
                         </div>
 
+                    </div>
+                    <div class="col-12 col-md-6 col-lg-3 mt-4">
+                        
+
+                            @*<option value="" disabled selected>Status</option>*@
+                            @Html.DropDownList("status", new List<SelectListItem> { new SelectListItem { Text = "Status", Value = "Status" },
```
Comment (bhairab15) [inline BookingSystem/Views/Dashboard/DashboardView.cshtml:126]: Why are we making hard coded item, can we use enum ?

File: BookingSystem/Controllers/DashboardController.cs [hunk 2]
```diff
@@ -95,6 +95,18 @@ public async Task<IActionResult> DashboardView(string txtHotel, string txtBookin
                 {
                     viewDashboard = viewDashboard.Where(s => s.Attributes.Customers.phone_number.Equals(txtContactNo)).ToList();
                 }
+                if (status != null)
+                {
+
+                    if (status == "All")
```
Comment (bhairab15) [inline BookingSystem/Controllers/DashboardController.cs:101]: Please use enum

File: BookingSystem/Controllers/DashboardController.cs [hunk 3]
```diff
@@ -43,7 +43,7 @@ public class DashboardController : Controller
         }
 
         //[HttpGet]
-        public async Task<IActionResult> DashboardView(string txtHotel, string txtBookingReferenceNo, string txtCustomerName, string txtContactNo)
+        public async Task<IActionResult> DashboardView(string txtHotel, string txtBookingReferenceNo, string txtCustomerName, string txtContactNo, string status)
         {
             _logger.LogInformation(nameof(BookingDashboardController) + "." + nameof(DashboardView) + Constant.start);
```
Comment (bhairab15) [inline BookingSystem/Controllers/DashboardController.cs:48]: Please use String interpolation

**Review by bhairab15 — CHANGES_REQUESTED** [review]

**Comment by Backiaraj07** [comment]
Comments resolved

**Pull request merged**